src/app/public/modules/sidebar/sidebar.component.ts

```diff
@@ -14,6 +14,7 @@ import {
 import {
   StacheRouteService
 } from '../router/route.service';
+import { StacheNavService } from '../nav/nav.service';
```

Comment:
Mind if we format the imports on new lines?
```suggestion

import {
  StacheNavService
} from '../nav/nav.service';
```

Reply:
Sure can.  Auto import and I forgot to fix it.  Sure we be nice if we could control the formatting of auto imports in VSCode.